### PR TITLE
Various cult 2.0 bugfixes

### LINKED ISSUE #19934: Can't print cult tomes in library
(WEB REPORT BY: jeanvoerman REMOTE: 172.93.109.202:7777)
# Revision
dc047cf4defee23b291035ad01053be0fa62816a

# Description
Emagged library computer to print false tomes, nothing showed up

# Steps to Reproduce
Emag library PC, try to print, receive nothing

# What you Expected
Cult tome

# What Actually Happened
Got the message it was printed, nothing appeared
"we're getting adminwarnings about them trying to be spawned when there's no cult so I guess not, you might wanna bug report it"

### PR DESCRIPTION
**Pull request by ShiftyRail**
Thanks to @iFlashYou for the reports
Features restored : 
- Emaging library computer : closes #19934
- Runes not costing blood
- Artificers being able to draw runes
- Being able to hide a stun paper in a book
- Supply talisman having more than one use for the communicate rune
- Nar-Sie name not being goofy in admin panel

### CODE REVIEW

File: code/datums/gamemode/factions/legacy_cult/talisman.dm [hunk 1]
```diff
@@ -164,7 +163,7 @@
 				var/obj/item/weapon/paper/talisman/T = new /obj/item/weapon/paper/talisman(get_turf(usr))
 				T.imbue = "emp"
 			if("conceal")
-				var/obj/item/weapon/paper/talisman/T = new /obj/item/weapon/paper/talisman(get_turf(usr))
+				var/obj/item/weapon/paper/talisman/T = new /obj/item/weapon/paper/talisman/communicate(get_turf(usr))
```
Comment (DamianX) [inline code/datums/gamemode/factions/legacy_cult/talisman.dm]: was this meant for line 169?

File: code/modules/library/lib_items.dm [hunk 2]
```diff
@@ -334,7 +334,7 @@
 			to_chat(user, "<span class='notice'>You carve out the pages from [title]! You didn't want to read it anyway.</span>")
 			carved = 1
 			return
-/*
+
 	else if(istype(W, /obj/item/weapon/paper/talisman))// TODO rolefix
```
Comment (DamianX) [inline code/modules/library/lib_items.dm]: what's to be done?

**Comment by ShiftyRail** [comment]
This is tested and it works.

**Comment by iFlashYou** [comment]
speedmerge

**Pull request merged**